### PR TITLE
Updated description of Weex

### PR DESCRIPTION
**Pull request by Jinjiang**
Ref: #1382
/ping @Hanks10100
Thanks.

### CODE REVIEW

File: src/v2/guide/comparison.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Since you don't need to know about JSX, ES2015, or build systems to get started
...
+React Native enables you to write native-rendered apps for iOS and Android using the same React component model. This is great in that as a developer, you can apply your knowledge of a framework across multiple platforms. On this front, Vue has an official collaboration with [Weex](https://weex.apache.org/), a cross-platform UI framework driven by Alibaba Group and being incubated to Apache Software Foundation (ASF), which uses Vue as its JavaScript framework runtime. This means with Weex, you can use the same Vue component syntax to author components that can not only be rendered in the browser, but also natively on iOS and Android!
```
Comment (Hanks10100) [inline src/v2/guide/comparison.md]: "... which uses Vue as one of its ..."

I think adding "one of" will be more accurate.

Comment (Jinjiang) [reply src/v2/guide/comparison.md]: Updated. Is that OK?

**Comment by Jinjiang** [comment]
Yes it already has another react-compatible framework called rax. https://github.com/alibaba/rax

**Comment by chrisvfritz** [comment]
@Jinjiang Thanks for clarifying. 🙂 I might want to revise that a bit, because from the wording it could be interpreted that Vue is one of the JS frameworks that _the ASF_ uses. 

Actually, I wonder if we might be complicating this section too much. What would you think if we just changed it to say Weex was _created_ by Alibaba Group? For example: 

> [...] On this front, Vue has an official collaboration with [Weex](https://weex.apache.org/), a cross-platform UI framework created by Alibaba Group. Weex allows you to use the same Vue component syntax to author components that can not only be rendered in the browser, but also natively on iOS and Android!

**Comment by Jinjiang** [comment]
That's OK for me. 👌
And I hope @Hanks10100 could have a check too.
Thanks

**Comment by Hanks10100** [comment]
Use "created" instead of "driven" looks good for me. But I think we should still mention the ASF, even if not mentioned Alibaba.

How many front-frameworks Weex support does not matter here in Vue's document. Mention it or not both fine.

**Comment by Jinjiang** [comment]
@chrisvfritz @Hanks10100 agree with both of you.
And because there are already some announcement about the incubation by ASF, so I preserve the ASF part description.
How about now? 🙂

**Pull request merged**